doc/nrf/releases/release-notes-1.9.0.rst

```diff
@@ -0,0 +1,845 @@
+
+  * :ref:`nrf_modem`:
+
+    * Updated:
+
+      * Architecture diagram in :ref:`nrfxlib:architecture`.
+      * :ref:`nrfxlib:nrf_modem_delta_dfu`.
+
+    * Removed:
+
+      * Overview diagram in :ref:`nrfxlib:nrf_modem`.
+      * AT socket page.
+      * Extensions page.
+
+Scripts
+=======
+
+This section provides detailed lists of changes by :ref:`script <scripts>`.
+
+Unity
+-----
+
+* Fixed a bug that resulted in the mocks for some functions not having the ``__wrap_`` prefix. This happened for functions declared with whitespaces between identifier and parameter list.
+
+HID Configurator for nRF Desktop
+--------------------------------
+
+* Updated:
+
+  * The script to recognize the bootloader variant as a DFU module variant for the configuration channel communication. The new implementation is backward compatible - the new version of the script checks for module name and acts accordingly. See the Performing DFU section in :ref:`nrf_desktop_config_channel_script` documentation.
+
+  * Documentation on setting up the dependencies for Debian/Ubuntu/Linux Mint operating system.
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``680ed07``, plus some |NCS| specific additions.
+
+The code for integrating MCUboot into |NCS| is located in the :file:`ncs/nrf/modules/mcuboot` folder.
+
+The following list summarizes both the main changes inherited from upstream MCUboot and the main changes applied to the |NCS| specific additions:
+
+* |no_changes_yet_note|
+
+Mcumgr
+======
+
+The mcumgr library contains all commits from the upstream mcumgr repository up to and including snapshot ``657deb65``.
+
+The following list summarizes the most important changes inherited from upstream mcumgr:
+
+* |no_changes_yet_note|
+
+Zephyr
+======
+
+.. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each NCS release.
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``3f82656``, plus some |NCS| specific additions.
+
+For a complete list of upstream Zephyr commits incorporated into |NCS| since the most recent release, run the following command from the :file:`ncs/zephyr` repository (after running ``west update``):
+
+.. code-block:: none
+
+   git log --oneline 3f82656 ^v2.6.0-rc1-ncs1
+
+For a complete list of |NCS| specific commits, run:
+
+.. code-block:: none
+
+   git log --oneline manifest-rev ^3f82656
+
+The current |NCS| main branch is based on the Zephyr v2.7 development branch.
+
+Matter (Project CHIP)
+=====================
+
+The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, ``77ab003e5fcd409cd225b68daa3cdaf506ed1107``.
```

Comment:
@Damian-Nordic , please check the commit hash.

Reply:
Seems correct.